Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,22 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string fName = GetInput("Enter your first name: ");
+            string lName = GetInput("Enter your last name: ");
+            int age = int.Parse(GetInput("Enter your age: "));
+            float height = float.Parse(GetInput("Enter your height (in cm): "));
+            float weight = float.Parse(GetInput("Enter your weight (in kg): "));
+            float bmi = weight / (height * height /10000);  // division to place decimal where it would be expected for a BMI measurement.
+            Console.WriteLine("{0} {1} is {2} years old, weighs {3} kg and is {4} cm tall.", fName, lName, age, weight, height);
+            Console.WriteLine("{0}'s BMI is {1}.", fName, bmi);
+            // I spent a week before this learning some C# and read Rob Mile's yellow book, so have applied some of what I learned here. 
```

Comment:
Yes, that's good! I can see that you apply thing that were not taught and make it a bit cleaner :)